### PR TITLE
Add initial files

### PR DESCRIPTION
**Pull request by ghost**
Added 

- migrations
- property models
- npm scripts

This PR relates to #2

### CODE REVIEW

**Review by Malnayr — APPROVED** [review]

File: src/models/property.ts [hunk 1]
```diff
@@ -0,0 +1,104 @@
...
+    .optional(),
+  check("address")
+    .isString()
+    .optional(),
+  check("description")
+    .isString()
+    .optional()
+];
+
+export function saveProperty(propertyPartial: Property) {
```
Comment (brett-larose) [inline src/models/property.ts:35]: I think `createProperty` would have made more sense here. Technically both creation and updating are "saving".

File: src/controller/property.ts [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+      return res.json({
+        property
+      });
+    })
+    .catch(next);
+}
+
+export function getProperty(req: Request, res: Response, next: NextFunction) {
+  return findProperty(req.params.propertyId)
+    .then(property => {
```
Comment (brett-larose) [inline src/controller/property.ts:49]: Shouldn't this check that the property isn't undefined/null and throw a 404 if it is?

Comment (ghost) [reply src/controller/property.ts:49]: Pretty sure the request won't get matched to this route I'd there's not properyId

Comment (brett-larose) [reply src/controller/property.ts:49]: The route just accepts any id though. So you could pass anything in and we would return null.

Comment (brett-larose) [reply src/controller/property.ts:49]: It's mostly just a pattern argument.

**Pull request merged**